### PR TITLE
added tutorial [MKC-593]

### PR DESCRIPTION
**Pull request by jhansson-ard**
## What This PR Changes
- Added missing tutorial

## Contribution Guidelines
- [x] I confirm that I have read the [contribution guidelines](https://github.com/arduino/docs-content/tree/main/contribution-templates) and comply with them.

### CODE REVIEW

File: content/hacking/01.software/PortManipulation/content.md [hunk 1]
```diff
@@ -0,0 +1,86 @@
...
+* Sometimes you might need to set multiple output pins at exactly the same time. Calling digitalWrite(10,HIGH); followed by digitalWrite(11,HIGH); will cause pin 10 to go HIGH several microseconds before pin 11, which may confuse certain time-sensitive external digital circuits you have hooked up. Alternatively, you could set both pins high at exactly the same moment in time using PORTB |= B1100;
+* If you are running low on program memory, you can use these tricks to make your code smaller. It requires a lot fewer bytes of compiled code to simultaneously write a bunch of hardware pins simultaneously via the port registers than it would using a for loop to set each pin separately. In some cases, this might make the difference between your program fitting in flash memory or not!
+
+### See
```
Comment (karlsoderby) [inline content/hacking/01.software/PortManipulation/content.md:84]:
```suggestion
### Read More
```

File: content/hacking/01.software/PortManipulation/content.md [hunk 2]
```diff
@@ -0,0 +1,86 @@
...
+
+PORTD is the register for the state of the outputs. For example;
+
+`PORTD = B10101000; // sets digital pins 7,5,3 HIGH` 
+
+You will only see 5 volts on these pins however if the pins have been set as outputs using the DDRD register or with pinMode().
+
+PIND is the input register variable It will read all of the digital input pins at the same time.
+
+#### Why use port manipulation?
```
Comment (karlsoderby) [inline content/hacking/01.software/PortManipulation/content.md:68]:
```suggestion
## Why Use Port Manipulation?
```

File: content/hacking/01.software/PortManipulation/content.md [hunk 3]
```diff
@@ -0,0 +1,86 @@
...
+
+DDRC - The Port C Data Direction Register - read/write
+
+PORTC - The Port C Data Register - read/write
+
+PINC - The Port C Input Pins Register - read only 
+
+Each bit of these registers corresponds to a single pin; e.g. the low bit of DDRB, PORTB, and PINB refers to pin PB0 (digital pin 8). For a complete mapping of Arduino pin numbers to ports and bits, see the diagram for your chip: [ATmega8](//www.arduino.cc/en/Hacking/PinMapping), [ATmega168](//www.arduino.cc/en/Hacking/PinMapping168). (Note that some bits of a port may be used for things other than i/o; be careful not to change the values of the register bits corresponding to them.)
+
+#### Examples
```
Comment (karlsoderby) [inline content/hacking/01.software/PortManipulation/content.md:44]:
```suggestion
## Examples
```

File: content/hacking/01.software/PortManipulation/content.md [hunk 4]
```diff
@@ -0,0 +1,86 @@
+---
+title: "Arduino - PortManipulation"
+source: "https://arduino.cc/en/Reference/PortManipulation"
+---
```
Comment (karlsoderby) [inline content/hacking/01.software/PortManipulation/content.md:4]: missing a description

File: content/hacking/01.software/PortManipulation/content.md [hunk 5]
```diff
@@ -0,0 +1,86 @@
...
+title: "Arduino - PortManipulation"
+source: "https://arduino.cc/en/Reference/PortManipulation"
```
Comment (jhansson-ard) [inline content/hacking/01.software/PortManipulation/content.md:3]:
```suggestion
source: "https://arduino.cc/en/Reference/PortManipulation"
description: 'Tutorial for manipulating pins on an Arduino Board'
```

Comment (jhansson-ard) [reply content/hacking/01.software/PortManipulation/content.md:3]: @karlsoderby does this description make sense?

File: content/hacking/01.software/PortManipulation/content.md (hunk 4 above)
Comment (karlsoderby) [inline content/hacking/01.software/PortManipulation/content.md:4]:
```suggestion
description: Advanced methods for how to port manipulation on an Arduino board.
---
```

**Review by karlsoderby — APPROVED** [review]

**Pull request merged**